.envrc

```diff
@@ -0,0 +1 @@
+use chefdk
```

Comment:
I would exclude this file completely; opens the door for other funky stuff like `.rvmrc` and we don't need that :)